Sources/BrowserServicesKit/Autofill/AutofillUserScript.swift

```diff
@@ -89,8 +91,10 @@ public class AutofillUserScript: NSObject, UserScript {
 
         switch message {
             case .emailHandlerStoreToken: return emailStoreToken
+            case .emailHandlerRemoveToken: return emailRemoveToken
```

Comment:
We should probably fix all these SwiftLint warnings here, even though they didn't originate in this PR. 🙂 Do you mind selecting this entire switch statement in Xcode then using `Control + I`? It'll re-indent the entire thing for you.